test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/listModels.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var path = require('path');
+var ModelConfig = app.models.ModelConfig;
+var chai = require('chai');
+var exampleWorkspace = path.resolve(__dirname, '../../../../../example');
+
+module.exports = function() {
+  var numberOfExpectedModels = 0;
+  var numberOfAvailableModels = 0;
+  var testcase = this;
```

Comment:
You didn't get rid of testcase, if you're going to keep it. I think we should rename it to "suite".